pipelinesScripts/validator/validator.go

```diff
@@ -257,6 +257,11 @@ func validateContent(descriptor *utils.PluginDescriptor) error {
 
 func runValidation(projectPath string) (err error) {
 	var output string
+	// Golang 1.16 or above requires to run 'go mod tidy' in order to update go.mod and go.sum files after the upgrade.
+	if output, err = utils.RunCommand(projectPath, true, "go", "mod", "tidy"); err != nil {
+		err = errors.New("Failed to run 'go mod tidy' plugin source code, located at " + projectPath + ". Error:\n" + output)
```

Comment:
The errro message seems to be phrased wrongly - "Failed to run 'go mod tidy' plugin source code..."